master/buildbot/data/base.py

```diff
@@ -53,8 +52,7 @@ def compileEventPathPatterns(self):
 
     def getEndpoints(self):
         endpoints = self.endpoints[:]
-        for i in range(len(endpoints)):
-            ep = endpoints[i]
+        for (i, ep) in enumerate(endpoints):
```

Comment:
I think parentheses around `i, ep` are usually omitted? That's what I see in other Python projects, and that's also how `enumerate()` is used in [PEP 279](https://www.python.org/dev/peps/pep-0279/), the PEP proposing `enumerate()`.

Reply:
OK I got rid of the parentheses next to the enumerate() statements.